README.md

```diff
@@ -149,11 +149,19 @@ type ThirdPartyStaticModuleRecord = {
   // Initializes the module if it is imported.
   // Initialize may return a promise, indicating that the module uses
   // the equivalent of top-level-await.
-  // XXX The compartment will leave that promise to dangle, so an eventual
+  // XXX The loader will leave that promise to dangle, so an eventual
   // rejection will necessarily go unhandled.
   initialize(environment: ModuleEnvironmentRecord, {
     import?: (importSpecifier: string) => Promise<ModuleExportsNamespace>,
-    importMeta?: Object
+    importMeta?: Object,
```

Comment:
I think this PR should base on #46? otherwise, there will be merge conflicts.

Reply:
I’ve landed #46. This is the base of a new stack of changes, though, so I’m doing my best to keep the conflicts under control.